src/main/java/org/five/sonarqubot/controllers/HttpRequests.java

```diff
@@ -0,0 +1,104 @@
+package org.five.sonarqubot.controllers;
+
+import org.apache.tomcat.util.codec.binary.Base64;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+
+@RestController
+public class HttpRequests {
+    @GetMapping("/")
+    public String helloWorld() {
+        return "Hello World! ";
+    }
+
+    @GetMapping("/search")
+    public ResponseEntity<Object> getProjects() {
+
+        RestTemplate restTemplate = new RestTemplate();
+        String url = "http://localhost:9000/api/projects/search";

```

Comment:
Would be nice to have application config value for:

http://localhost:9000/api

(since it might change)